File: support_tools/swigwin-3.0.12/Examples/test-suite/go/cpp_enum_runme.go
package main

import "./cpp_enum"

func main() {
	f := cpp_enum.NewFoo()

	if f.GetHola() != cpp_enum.FooHello {
		panic(f.GetHola())
	}

	f.SetHola(cpp_enum.FooHi)
	if f.GetHola() != cpp_enum.FooHi {
		panic(f.GetHola())
	}

	f.SetHola(cpp_enum.FooHello)

	if f.GetHola() != cpp_enum.FooHello {
		panic(f.GetHola())
	}

	cpp_enum.SetHi(cpp_enum.Hello)
	if cpp_enum.GetHi() != cpp_enum.Hello {
		panic(cpp_enum.Hi)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/cpp_enum_runme.py
import cpp_enum

f = cpp_enum.Foo()

if f.hola != f.Hello:
    print f.hola
    raise RuntimeError

f.hola = f.Hi
if f.hola != f.Hi:
    print f.hola
    raise RuntimeError

f.hola = f.Hello

if f.hola != f.Hello:
    print f.hola
    raise RuntimeError

cpp_enum.cvar.hi = cpp_enum.Hello
if cpp_enum.cvar.hi != cpp_enum.Hello:
    print cpp_enum.cvar.hi
    raise RuntimeError
